docs/source/distributions.rst

```diff
@@ -6,23 +6,30 @@ Distributions
    :maxdepth: 2
    :caption: Contents:
 
-Primitive Distributions
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
 PyTorch Distributions
----------------------
+~~~~~~~~~~~~~~~~~~~~~
+
+Most distributions in Pyro are thin wrappers around PyTorch distributions.
+For details on the PyTorch distribution interface, see
+:class:`torch.distributions.Distribution`.
+For differences between the Pyro and PyTorch interfaces, see
+:class:`pyro.distributions.torch_distribution.TorchDistributionMixin`.
 
 .. automodule:: pyro.distributions.torch
     :members:
     :undoc-members:
-    :show-inheritance:
+    :special-members: __call__
+
+Primitive Distributions
```

Comment:
Is it okay to have multivariate normal implementations under Primitive Distributions, or should we create a header like `Pyro Distributions` as a parking space for distributions that are not yet migrated to PyTorch (or better kept in Pyro like the Delta distribution)?